contracts/mainnet/connectors/beta/main.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^0.7.0;
+
+/**
+ * @title Authority.
+ * @dev Manage Authorities to DSA.
+ */
+
+import { AccountInterface } from "../../common/interfaces.sol";
+import { Events } from "./events.sol";
+
+abstract contract AuthorityResolver is Events {
+    /**
+     * @dev Enable beta mode
+     * @notice enabling beta mode gives early access to new/risky features
+     */
+    function enable() external payable returns (string memory _eventName, bytes memory _eventParam) {
+        AccountInterface _dsa = AccountInterface(address(this));
+        require(!_dsa.isBeta(), "beta-already-enabled");
+        _dsa.toggleBeta();
+
+        _eventName = "LogEnableBeta()";
```

Comment:
@thrilok209 not encoding `_eventParam` as it doesn't have any inputs. Will it work fine?

Reply:
Yes! It will work fine. By default it will be `0x`.